nixos/lib/test-driver/test-driver.py

```diff
@@ -980,18 +984,50 @@ def subtest(name: str) -> Iterator[None]:
         help="re-use a VM state coming from a previous run",
         action="store_true",
     )
-    (cli_args, vm_scripts) = arg_parser.parse_known_args()
+    arg_parser.add_argument(
+        "-I",
+        "--interactive",
+        help="drop into a python repl and run the tests interactively",
+        action="store_true",
+    )
+    arg_parser.add_argument(
+        "--start-scripts",
+        action=EnvDefault,
+        envvar="startScripts",
+        nargs="*",
+        help="start scripts for participating virtual machines",
+    )
+    arg_parser.add_argument(
+        "--vlans",
+        action=EnvDefault,
+        envvar="vlans",
+        nargs="*",
+        help="vlans to span by the driver",
+    )
```

Comment:
when running the script with `--help`, it explains this parameter like `--clans [VLANS [VLANS ...]]`.
This looks a bit misleading, as it reads like each placeholder keeps a list of vlans. it might be more comprehensible if it was like `--vlans [VLAN [VLAN ...]]`.

The same applies to `--start-scripts`. It might be like a little nitpick here, but if there is a way to make the help text more comprehensible, we should do that.

Reply:
`metavar` argument to `arg_parser.add_argument` fortunately does exactly this. :smile: